site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.tsx

```diff
@@ -218,9 +222,21 @@ export const CreateWorkspacePageView: FC<CreateWorkspacePageViewProps> = ({
 		<Margins size="medium">
 			<PageHeader
 				actions={
-					<Button size="sm" variant="outline" onClick={onCancel}>
-						Cancel
-					</Button>
+					<Stack direction="row" spacing={2}>
```

Comment:
the `actions` prop is already rendered inside of a flex container that adds spacing. just use a fragment.